File: tests/fixtures/java/CalcTest.java
public final class CalcTest {
    public static void main(String[] args) {
        assertFalse(Calc.isBig(2));
        assertTrue(Calc.isBig(3));
    }

    private static void assertTrue(boolean value) {
        if (!value) {
            throw new AssertionError("expected true");
        }
    }

    private static void assertFalse(boolean value) {
        if (value) {
            throw new AssertionError("expected false");
        }
    }
}

File: tests/fixtures/csharp/CalcTest.cs
public static class CalcTest
{
    public static void Main()
    {
        AssertFalse(Calc.IsBig(2));
        AssertTrue(Calc.IsBig(3));
    }

    private static void AssertTrue(bool value)
    {
        if (!value)
        {
            throw new System.Exception("expected true");
        }
    }

    private static void AssertFalse(bool value)
    {
        if (value)
        {
            throw new System.Exception("expected false");
        }
    }
}
